service/src/test/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiControllerBqDatasetTest.java

```diff
@@ -128,10 +132,20 @@ public void create() throws Exception {
     assertEquals(
         expectedBqDataset.getAttributes().getProjectId(),
         actualBqDataset.getAttributes().getProjectId());
+    ActivityLogChangeDetails changeDetails =
```

Comment:
Let's not put individual field assertions in these methods. I purposely used helper methods to keep these tests short and readable.

Advantages of helper methods:

- Tests are shorter and easier to read (tens vs hundreds of lines). When there are 10 slightly different tests, easier to tell differences apart.
- Instead of adding assertions to dozens of tests, you just add to one helper method.

Let's add these assertions to assertResourceMetadata (which is now in MockMvcUtils.java).

opt Also, please assert the entire row, not just these new columns. I forgot to ask for that in the last PR.